src/docs/Foundations/Typography/BodyStyles/index.mdx

```diff
@@ -1,27 +1,46 @@
 ---
 title: 'Body styles'
 order: 4
+description: 'A body is a paragraph or a multiline text of a pattern or a page. It must provide basic typographic styles like font-size, line-height and font-weight.'
 status:
   sketch: 'wip'
   scss: 'wip'
 ---
 
-> The body patterns provide basic typographic styles for readable multilines text or paragraphs like a product description or an article.
+> A body is a paragraph or a multiline text of a pattern or a page. It must provide basic typographic styles like font-size, line-height and font-weight.
 
-Body styles are patterns meant to help you use consistent typographic hierarchy with the context and across responsive viewports. It's mainly reserved for users wishing to focus on tasks with added value and user experience. Text classes are families of several font-sizes and help create a hierarchy of contents.
+## Recommended size
 
-#### Body styles are defined with three factors :
+### Body Medium
 
-- Font-size _(that may vary depending on the viewport)_
-- Line-height _(that are optimised for readability and accessibility)_
-- Font-weight _(regular or semi-bold)_
+The default body style is **body medium** (`mt-body-m`) with the following specifications:
 
-#### And comes with two variations for each classe :
+| Font-size           | Font-weight           | Line-height           |
+| ------------------- |---------------------- | --------------------- |
+| 16px                | Regular               | `$line-height m`      |
 
-Each body pattern comes with a modifier to set the font-weight to **'semi bold'** and a modifier to use an alternative **line-height**.
+<Preview path="typographyBodyMedium" nude />
 
-|                                         |                                      |
-| --------------------------------------- | ------------------------------------ |
-| <Preview path="typographyBodySmall" />  | **Body Small** <br /> 14px (0.875mu) |
-| <Preview path="typographyBodyMedium" /> | **Body Medium** <br /> 16px (1mu)    |
-| <Preview path="typographyBodyLarge" />  | **Body Large** <br /> 18px (1.125mu) |
+## Specific needs
+
+Mozaic's typographic system still allows you to answer specific needs in some cases like smaller or larger font sizes and also thicker font weight like **semibold**.
+
+### Body Small
+
+For accessibility and readability, this is the smallest value you can use for this kind of text.
```

Comment:
What is this kind of text ? We should be specific. 
for multiline text ? 
what about legal mentions ? should we add another size or should they be 14px, or is it another pattern ? Because the definition of a legal mention match `multiline text or paragraphs`

Reply:
yes it is for multiline texts, I'll detail that in the sentence.

for legal mentions these are not considered as body styles. They can use 12px for mentions